### ISSUE TITLE
ObjectJob is always null when transferring domains

### ISSUE THREAD

**Issue by andlinger** [InterNetX/java-domainrobot-sdk#21]
See #14 

When transferring domains, the result, the ObjectJob, contains only null objects:

```
class ObjectJob {
    job: null
    object: null
    niccomLogs: null
    authentication: null
    dcvAuth: null
}
```

When looking at the code, the error might also occur in the methods DomainClient.update, DomainClient.renew, DomainClient.updateStatus and DomainClient.restore.

**Comment by Ephenodrom** [comment]
Hello @andlinger 
Yes this is the same problem as mentioned in #14. I updated the official swagger documentation. I will release a new version today and this should automatically fix this problem, due to the models are now generated correctly.

**Comment by andlinger** [comment]
@Ephenodrom Could you also create a new version of the SDK?

**Comment by Ephenodrom** [comment]
Hello @andlinger 
The new release is ready and this is fixed with version 0.2.7.

Regards
